templates/about2expire.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+$this->data['header'] = sprintf(
+    'Your password will expire in %s %s',
+    $this->data['daysleft'],
+    $this->data['dayOrDays']
+);
+$this->data['autofocus'] = 'yesbutton';
+
+$this->includeAtTemplateBase('includes/header.php');
+
+?>
+<p>
+  The password for your <?= htmlentities($this->data['accountName']); ?>
```

Comment:
Yes, that can be used here. The previous version had been using that translate function, but it was overly complicated and was only offering English and Slovenian.